server/app/models/person.rb

```diff
@@ -24,6 +24,7 @@ class Person < ActiveRecord::Base
   has_attached_file :image
 
   has_many :castings, dependent: :destroy
+  belongs_to :listable, polymorphic: true
```

Comment:
Yeah, that's generally how people refer to the ad-hoc grouping of things which can fit into a specific polymorphic association, but in this case you're not referring to the association, you're referring to the list items — different direction, different mane